src/components/atoms/Icon/icons/Add.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+export const Add = {
+  viewBox: '0 0 18 18',
+  jsx: (
```

Comment:
zmieniłbym nazwę jsx eksportowaną z ikon na coś w stylu iconBody albo svgBodyJsx - nazwa jsx sugeruje wrzucanie tu dowolnej wartości będącej jsx'em i choć zadziała to czy tego chcemy?

Reply:
ja bym został przy `jsx`. Jest krócej, dodajemy plik w folderze icons więc nie wiem za bardzo co tam mielibyśmy innego dodawać jak nie kawałek svg